packages/contracts/test/integration/commitReveal.spec.ts

```diff
@@ -104,27 +82,25 @@ describe("CommitRevealApp", async () => {
   it("should complete a full lifecycle", async function() {
     this.timeout(4000);
 
-    const startBalanceA = await A.getBalance();
-    const startBalanceB = await B.getBalance();
+    const startBalanceA = await alice.getBalance();
+    const startBalanceB = await bob.getBalance();
+
     // 1. Deploy & fund multisig
-    const multisig = new Multisig([A.address, B.address]);
+    const multisig = new Multisig([alice.address, bob.address]);
     await multisig.deploy(masterAccount);
     await masterAccount.sendTransaction({
       to: multisig.address,
-      value: Utils.UNIT_ETH.mul(2)
+      value: ethers.utils.parseEther("2")
     });
 
     const terms = {
       assetType: AssetType.ETH,
-      limit: Utils.UNIT_ETH.mul(2),
-      token: Utils.zeroAddress
+      limit: ethers.utils.parseEther("2"),
+      token: ZERO_ADDRESS
     };
     // 2. Deploy CommitRevealApp app
-    const appContract = await Utils.deployContract(
-      CommitRevealApp,
-      masterAccount
-    );
-    app = {
+    const appContract = await deployContract(CommitRevealApp, masterAccount);
+    const app = {
```

Comment:
We could write a type for this. addr is a 20 byte string, reduce could be a ethers.Interface.FunctionDefinition etc

Reply:
Already on it